lib/src/base-styles/_nx-page-header.scss

```
@@ -41,61 +41,60 @@ $main-header-product-grid-rows: $main-header-branding-height / 2;
   color: $nx-off-white;
   display: -ms-grid;
   display: grid;
-  grid-column-gap: 2px;
-  grid-row-gap: 2px;
-  -ms-grid-rows: $main-header-product-grid-rows 2px auto;
-  grid-template-rows: $main-header-product-grid-rows auto;
-  -ms-grid-columns: $main-header-branding-height 2px auto;
-  grid-template-columns: $main-header-branding-height auto;
+  grid-column-gap: $nx-spacing-xxs;
+  -ms-grid-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
+  grid-template-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
+  -ms-grid-columns: $main-header-product-height $nx-spacing-xxs auto;
+  grid-template-columns: $main-header-product-height auto;
   grid-template-areas:
     "logo title"
     "logo version";
-  height: $main-header-branding-height;
+  height: $main-header-product-height;
   margin-top: 10px;
   padding-right: 12px;
   text-decoration: none;
   min-width: $nx-sidebar-width;
 }
 
 .nx-product__logo {
+  -ms-grid-row: 1;
+  -ms-grid-row-span: 2;
+  -ms-grid-column: 1;
   -ms-grid-row-align: center;
   align-self: center;
   box-sizing: border-box;
-  -ms-grid-row: 1;
-  -ms-grid-row-span: 3;
-  -ms-grid-column: 1;
   grid-area: logo;
-  height: $main-header-branding-height;
-  width: $main-header-branding-height;
+  height: $main-header-product-height;
+  width: $main-header-product-height;
```

```suggestion
  height: 100%;
  width: 100%;
```
(Only works correctly in conjunction with my `fr` suggestion above, otherwise half a pixel of height is lost to rounding)